internal/store/service.go

```diff
@@ -184,6 +185,28 @@ func serviceMetricFamilies(allowAnnotationsList, allowLabelsList []string) []gen
 				}
 			}),
 		),
+		*generator.NewFamilyGeneratorWithStability(
+			"kube_service_address",
```

Comment:
on second thought i think its better to use this name like they did for endpoint:
- kube_service_ports